validator/src/cli.rs

```diff
@@ -1375,6 +1375,24 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 )
                 .hidden(hidden_unless_forced()),
         )
+        .arg(
+            Arg::with_name("accounts_db_ancient_ideal_storage_size")
```

Comment:
I find "size" a bit confusing with "file size". 
Here I think it is intended for the number of ancient storage. How about rename it to "accounts_ideal_db_number_of_ancient_storages"?

Reply:
I think this is actually size as in bytes of the files that will be created. The alive_bytes of accounts are accumulated and compared against this value.

Reply:
oh. you are right.